ignf_gpf_api/workflow/Workflow.py

```diff
@@ -49,13 +50,14 @@ def get_raw_dict(self) -> Dict[str, Any]:
         """
         return self.__raw_definition_dict
 
-    def run_step(self, step_name: str, callback: Optional[Callable[[ProcessingExecution], None]] = None, behavior: Optional[str] = None) -> List[StoreEntity]:
+    def run_step(self, step_name: str, callback: Optional[Callable[[ProcessingExecution], None]] = None, behavior: Optional[str] = None, datastore: Optional[str] = None) -> List[StoreEntity]:
         """Lance une étape du workflow à partir de son nom. Liste les entités créées par chaque action et le retourne.
 
         Args:
             step_name (str): nom de l'étape
-            callback (Optional[Callable[[ProcessingExecution], None]], optional): callback de suivi si création d'une exécution de traitement.
-            behavior (Optional[str]): comportement à adopter si une entité existe déjà sur l'entrepôt.
+            callback (Optional[Callable[[ProcessingExecution], None]], optional): callback de suivi si création d'une exécution de traitement. Defaults to None.
+            behavior (Optional[str]): comportement à adopter si une entité existe déjà sur l'entrepôt. Defaults to None.
+            datastore (Optional[str]): id du datastore à utiliser. Si None, le datastore sera Le premier trouvé dans l'action puis dans workflow puis dans configuration. Defaults to None.
```

Comment:
Il ne faut mieux pas mettre les "Default to ..." : ils sont ajoutés par l'outil de génération de documentation HTML, donc ça fait doublon.

```suggestion
            callback (Optional[Callable[[ProcessingExecution], None]], optional): callback de suivi si création d'une exécution de traitement.
            behavior (Optional[str]): comportement à adopter si une entité existe déjà sur l'entrepôt.
            datastore (Optional[str]): id du datastore à utiliser. Si None, le datastore sera le premier trouvé dans l'action puis dans workflow puis dans configuration.
```